cmd/sts-handlers.go

```diff
@@ -190,6 +190,18 @@ func parseForm(r *http.Request) error {
 	return nil
 }
 
+// getTokenSigningKey returns secret key used to sign JWT session tokens
+func getTokenSigningKey() (string, error) {
+	secret := globalActiveCred.SecretKey
+	if globalSiteReplicationSys.isEnabled() {
+		if !globalSiteReplicatorCred.IsValid() {
```

Comment:
Just like we do loadUser() we must try reloading this from disk and then throw an error if any.